portfolio/src/main/webapp/index.html

```diff
@@ -26,9 +26,19 @@ <h1>Tal's Portfolio</h1>
  
       <p>Post a comment here!</p>
       <form method="POST" action="/data">
-      <input type="text" name="text" />
-      <br/>
-      <button id = "button">Submit Comment</button>
+        <input type="text" name="text" />
+        <br/>
+        <button id = "button">Submit Comment</button>
+      </form>
+    
+      <p>How many comments should display:</p>
+     
+     
+      <form method="POST" action="/data">
+        <input type="number" name="choice" min="0" max="15" value="0">
+        <br/><br/>
+         <!-- shouldn't load comments until clicked but this isn't working -->
+        <button id = "button" onClick="loadComments">Submit #</button>
```

Comment:
Now, shouldn't loadComments() only happen after I click the submit number of comments button?